fstat/static/js/snackbar.js

```diff
@@ -0,0 +1,17 @@
+$("body").prepend('<div id="snackbar"></div>')
```

Comment:
What if we use snackbar.js in multiple pages? In that case we will need to add that thing to all pages which I feel is inconvenient.